programming.md

```diff
@@ -50,6 +50,17 @@ Website - https://www.udacity.com/course/intro-to-html-and-css--ud001
 [![html and CSS course by udacity](https://user-images.githubusercontent.com/4047597/66047080-c3619580-e544-11e9-8d37-d478e778ed98.png)](https://www.udacity.com/course/intro-to-html-and-css--ud001)
 
 
+## Java
+
+### Java for Complete Beginners - John Purcell on Udemy
+
+This is the most popular free Java course on Udemy. If you are a beginner in Java, go for this. It offers a thorough introduction to every Java concept and is taught in digestible sessions.
+
+Website - https://www.udemy.com/course/java-tutorial/
+
+![Java for Complete Beginners - John Purcell](https://user-images.githubusercontent.com/51238282/66408079-e8a24800-ea0b-11e9-916e-658dc4848039.png)
```

Comment:
Thanks. This PR looks solid except that we need to link the image to the URL as well.

Reply:
Opps, missed it!
Kindly review the latest change :- https://github.com/aviaryan/learn-for-free/pull/8/commits/5034bc8e6600e1285c6952e35d193ef0e8824586